rts/Makefile

```diff
@@ -2,8 +2,12 @@ ifeq ($(DEBUG),1)
    EXTRA_OPTS = -g -DDEBUG -O0
 endif
 
+ifeq ($(NO_GC),1)
+   GC_OPTS = -DNO_GC
+endif
+
 CC ?= clang
-CFLAGS ?= -Wall -Wextra -Werror -O2 -Iinclude $(EXTRA_OPTS)
+CFLAGS ?= -Wall -Wextra -Werror -O2 -Iinclude $(EXTRA_OPTS) $(GC_OPTS)
```

Comment:
I think we could now rename EXTRA_OPTS --> DEBUG_OPTS for consistency.

Reply:
Yeah, that sounds about right